test/RPM/SpecFileSpec.hs

```diff
@@ -80,11 +80,11 @@ spec = do
       getTypeFromLine "Requires: xz = 1" `Test.shouldBe` Just (RuntimeRequires $ mkVerDep "xz" $ CEq "1")
   --
   Test.describe "rpm dependency grapher" $
-    Test.it "should produce flat BuildRequires" $ do
+    Test.it "should produce flat RuntimeRequires" $ do
       let deps = Dependencies [boostDep, xzDep] [tacpDep, jsoncppDep]
           gr = buildGraph deps
 
-      expectDeps (map toDependency [boostDep, xzDep]) gr
-      expectDirect (map toDependency [boostDep, xzDep]) gr
+      expectDeps [toDependency tacpDep] gr
+      expectDirect [toDependency tacp] gr
```

Comment:
This should include both of the runtime deps.

on [line 84](https://github.com/fossas/spectrometer/pull/113/files#diff-8f235742309df0fca599b6da5b06d997R84), we have the creation of the base structure as `Dependencies buildRequiresList runtimeRequiresList`.

This should produce a flat graph (planar?) equal to the list passed in the constructor.

I would consider reverting this part of the change and just changing the variables to the other list of deps:

```suggestion
      expectDeps (map toDependency [tacpDep, jsoncppDep]) gr
      expectDirect (map toDependency [tacpDep, jsoncppDep]) gr
```